include/commissioner/network_diag_data.hpp

```diff
@@ -145,10 +146,78 @@ struct Connectivity
     uint8_t  mLeaderCost              = 0;
     uint8_t  mIdSequence              = 0;
     uint8_t  mActiveRouters           = 0;
-    uint16_t mRxOffChildBufferSize    = 0;
```

Comment:
Is this an intended removal?

If yes, `kRxOffChildBufferSizeBit` should be removed too